templates/base/graphQl/ApolloClient.js

```diff
@@ -2,17 +2,18 @@ import ApolloClient, { HttpLink, InMemoryCache } from 'apollo-boost';
 import { ApolloLink } from 'apollo-link';
 import { onError } from 'apollo-link-error';
 import { graphQLApiUri } from '../config/endpoints';
+import Logger from '../src/utils/Logger';
 
 const cache = new InMemoryCache();
 
 const errorLink = onError(({ graphQLErrors, networkError }) => {
```

Comment:
{ graphQLErrors, networkError } also gives you operationName as one of the parameter. That contain the query name that failed

Reply:
done.